test/petab/test_amici_prediction.py

```diff
@@ -0,0 +1,292 @@
+        s_outs1[:, :, 3] = amici_sy[0][:, 1, :] \
+            / np.tile(amici_y[0][:, 0], (2, 1)).transpose()
+        s_outs1[:, :, 4] = amici_sy[0][:, 1, :] \
+            / np.tile(amici_y[0][:, 0], (2, 1)).transpose()
+
+        s_outs2 = np.zeros((10, 2, 5))
+        s_outs2[:, :, 0] = amici_sy[0][:, 1, :] \
+            / np.tile(amici_y[0][:, 0], (2, 1)).transpose()
+        s_outs2[:, :, 1] = amici_sy[0][:, 1, :] \
+            / np.tile(amici_y[1][:, 0], (2, 1)).transpose()
+        s_outs2[:, :, 2] = amici_sy[1][:, 1, :] \
+            / np.tile(amici_y[1][:, 0], (2, 1)).transpose()
+        s_outs2[:, :, 3] = amici_sy[2][:, 1, :] \
+            / np.tile(amici_y[1][:, 0], (2, 1)).transpose()
+        s_outs2[:, :, 4] = amici_sy[2][:, 1, :] \
+            / np.tile(amici_y[2][:, 0], (2, 1)).transpose()
+        return [s_outs1, s_outs2]
+
+    def ppt_out(raw_outputs):
+        amici_t = [raw_output['t'] for raw_output in raw_outputs]
+        # compute ratios of simulations across conditions
+        t_out1 = amici_t[0]
+        t_out2 = amici_t[1]
+
+        return [t_out1, t_out2]
+
+    # get the model and the edatas, create an objective
+    model, solver, edatas = edata_objects
+    objective = pypesto.AmiciObjective(model, solver, edatas, 1)
+    # now create a prediction object
+    complex_prediction = pypesto.AmiciPredictor(
+        objective, max_chunk_size=2, post_processor=pp_out,
+        post_processor_sensi=pps_out, post_processor_time=ppt_out,
+        observable_ids=[f'ratio_{i_obs}' for i_obs in range(5)])
+    # let's set the parameter vector
+    x = np.array([3., 0.5])
+
+    # assert output is what it should look like when running in efault mode
+    p = complex_prediction(x, sensi_orders=(0, 1))
+    check_outputs(p, out=(0, 1), n_cond=2, n_timepoints=10, n_obs=5, n_par=2)
+
+    # assert folder is there with all files
+    # remove file is already existing
+    if os.path.exists('deleteme'):
+        shutil.rmtree('deleteme')
+    p = complex_prediction(x, output_file='deleteme.csv', sensi_orders=(0, 1),
+                           output_format='csv')
+    check_outputs(p, out=(0, 1), n_cond=2, n_timepoints=10, n_obs=5, n_par=2)
+    # check created files
+    assert os.path.exists('deleteme')
+    expected_files = {'deleteme_0.csv', 'deleteme_0__s0.csv',
+                      'deleteme_0__s1.csv', 'deleteme_1.csv',
+                      'deleteme_1__s0.csv', 'deleteme_1__s1.csv'}
+    assert set(os.listdir('deleteme')) == expected_files
+    shutil.rmtree('deleteme')
+
+    # assert h5 file is there
+    p = complex_prediction(x, output_file='deleteme.h5', sensi_orders=(0, 1),
+                           output_format='h5')
+    check_outputs(p, out=(0, 1), n_cond=2, n_timepoints=10, n_obs=5, n_par=2)
+    assert os.path.exists('deleteme.h5')
+    os.remove('deleteme.h5')
+
+
+def test_petab_prediction():
+    yaml_file = '../../doc/example/conversion_reaction/' \
```

Comment:
Please use relative path based on `__file__` to make tests run from different working directories.